### ISSUE TITLE
Pre-defined aliases not required when using babashka nrepl

### ISSUE THREAD

**Issue by uosl** [liquidz/vim-iced#206]
Thanks for adding support for babashka nrepl so quickly!

I'm using it like the following
```
$ bb --nrepl-server 5555
:IcedConnect 5555
```
and it works as expected, except that the aliases listed in the [babashka readme](https://github.com/borkdude/babashka/blob/master/README.md#usage) that are available through pre-defined aliases have to be required manually (ie. `(require '[clojure.string :as str])` and so on).

Would it be possible to have these required by default when using vim-iced?

**Comment by liquidz** [comment]
@uosl Thanks for your reporting!

I couldn't reproduce the problem with following steps:
* `echo '(comment (str/trim "  foo bar  "))' > foo.clj`
* `bb --nrepl-server 5555`
* `vim foo.clj -c 'IcedConnect 5555'`
* Evaluate the `comment` form

Are you sure that you have `ns` form in your file?

Babashka pre-defines some aliases to `user` namespace, and vim-iced will load current file when you connected to nREPL.
So if you have a `ns` form, your `*ns*` will be changed from `user` to another.

**Comment by uosl** [comment]
You're right, I restarted nvim and everything and it's working fine.

I remember now that when I had this problem, evaluating defs would return `#'sci.core/,,,` or something along those lines, and I'm not sure how I managed to change into that namespace. (I checked the file history and I never called any ns functions.) I'm guessing I mistyped some hotkey.

Anyways, thanks for checking and sorry for the false report!
